### ISSUE TITLE
Void bee don't produce Void Metal Nuggets

### ISSUE THREAD

**Issue by KorDum** [MagicBees/MagicBees#149]
Void bee don't produce Void Metal Nuggets, but produce Honey Combs.
NEI Addons show production: Honey Comb (10%) and Void Metal Nugget (15.5%). Is a bug?

**Comment by yznpku** [comment]
Void metal nugget is the spec produce. Are you sure you put the bee in the correct biome?

**Comment by KorDum** [comment]
Is Magic Forest required?

**Comment by yznpku** [comment]
iirc it requires T:Cold/H:Normal, so try to put it in the End?

**Comment by KorDum** [comment]
No, it's not that, it's T:Normal/H:Normal after breeding and it is work.
